models/staging/commissioning/sus/stg_ecds_consolidated.sql

```
@@ -0,0 +1,327 @@
+,Co."2" AS "Comorbidity2" 
+,Co."3" AS "Comorbidity3" 
+,Co."4" AS "Comorbidity4"
+,Co."5" AS "Comorbidity5"
+,Co."6" AS "Comorbidity6" 
+,Co."7" AS "Comorbidity7" 
+,Co."8" AS "Comorbidity8" 
+,Co."9" AS "Comorbidity9" 
+,Co."10" AS "Comorbidity10" 
+
+--Mental Health Legal Status Field 
+,'MH.CLASSIFICATION' AS "Mental_Health_Legal_Status_Classification" 
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)